.github/workflows/website.yml

```diff
@@ -37,9 +37,11 @@ jobs:
         with:
           fetch-depth: 0   # Fetch all history for .gitinfo and .lastmod
       - name: "Generate site using antora site action"
-        uses: kameshsampath/antora-site-action@master
+        uses: fsismondi/antora-site-action@master
```

Comment:
You need at least to fork into Camptocamp

Reply:
Has there been a PR sent upstream to integrate your changes?

Reply:
https://github.com/kameshsampath/antora-site-action/pull/6 
\o/

Ill proceed with the updates on our side